Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -221,6 +221,54 @@ class PaymentSheetUITest: XCTestCase {
         XCTAssertNotNil(successText.label.range(of: "Your order is confirmed!"))
     }
 
+    func testDeferredPaymentIntent() {
```

Comment:
With these tests, are the snapshot tests you added earlier still useful?

Reply:
Yeah, these are essentially snapshot tests now but really they will be confirm tests.

If these tests make the snapshot tests not useful, then I would imagine that would apply for many of the other snapshots as well.

Reply:
It might, yeah!  The way I'm thinking about it, snapshot tests are for testing UI, and deferred workflow is unrelated to UI, so we don't need snapshot tests for it.

Reply:
The way I viewed those snapshot tests were "We correctly render PaymentSheet UI when provided an `IntentConfiguration`". These automated UI tests may miss subtle unintended changes to the UI as they only really look for specific buttons/text that are critical to the UI flow. It seems plausible that a UI bug could crop up only when loading from a deferred intent/intent config that would be missed during automated tests since we don't look at every UI element on screen. The snapshot gives us 100% coverage of what we expect the UI to look like when loaded from an intent config while these do not.